core/src/main/java/org/elasticsearch/index/query/FuzzyQueryParser.java

```diff
@@ -63,15 +51,12 @@ public Query parse(QueryParseContext parseContext) throws IOException, QueryPars
 
         String value = null;
         float boost = 1.0f;
-        Fuzziness fuzziness = DEFAULT_FUZZINESS;
-        int prefixLength = FuzzyQuery.defaultPrefixLength;
-        int maxExpansions = FuzzyQuery.defaultMaxExpansions;
-        boolean transpositions = false;
+        Fuzziness fuzziness = FuzzyQueryBuilder.DEFAULT_FUZZINESS;
+        int prefixLength = FuzzyQueryBuilder.DEFAULT_PREFIX_LENGTH;
+        int maxExpansions = FuzzyQueryBuilder.DEFAULT_MAX_EXPANSIONS;
+        boolean transpositions = FuzzyQueryBuilder.DEFAULT_TRANSPOSITIONS;
+        String rewrite = null;
         String queryName = null;
-        MultiTermQuery.RewriteMethod rewriteMethod = null;
-        if (parseContext.isFilter()) {
-            rewriteMethod = MultiTermQuery.CONSTANT_SCORE_REWRITE;
-        }
```

Comment:
OK we should only rewrite if we have a filter or if `rewrite` is explicitly set, otherwise the default rewrite on  FuzzyQuery is untouched. If we want to make the `rewrite` options as constants, we would have to do so for every query that supports `rewrite`. Not sure this is really necessary.